app/views/clients/_note_form.haml

```diff
@@ -2,4 +2,5 @@
 = simple_form_for @note, as: :note, url: client_notes_path(@client) do |f|
   -# = f.input :type, label: 'Note type', collection: GrdaWarehouse::ClientNotes::Base.available_types(), label_method: :type_name
   = f.input :note, label: 'Note'
+  = f.hidden_field :type, value: 'GrdaWarehouse::ClientNotes::ChronicJustification'
```

Comment:
This doesn't look right to me.  Why hard-code the type?

Reply:
That partial gets displayed on the `Chronic` tab, which says `Add Chronic Justification`. This form was broken because no type was being passed to the notes controller. This fixes it.

Reply:
Are we sure this isn't in use elsewhere?  Also, can we just hard-code the type in the controller?  It'd be safer.

Reply:
I did a search, its only being used there and yes i can move the type to the controller

Reply:
Thanks.

Reply:
actually the type is being passed in on another form (client tab -> notes) so the controller is expecting that parameter and for that form it can be one of two different types of notes. I can use different strings like 'chronic' and 'window' if you don't like passing in the namespaced classname.

Reply:
That makes sense.  Let's leave it.  We need to make sure we're whitelisting the types in the controller so you can't send some arbitrary string.

Reply:
an error gets raised in the create action if the type is not found in `GrdaWarehouse::ClientNotes::Base.available_types`

Reply:
Perfect